{
  "cve": {
    "id": "CVE-2020-24979",
    "sourceIdentifier": "cve@mitre.org",
    "vulnStatus": "Rejected",
    "published": "2020-09-04T00:15:10.833",
    "lastModified": "2023-11-07T03:20:11.273",
    "descriptions": [
      {
        "lang": "en",
        "value": "Rejected reason: DO NOT USE THIS CANDIDATE NUMBER. ConsultIDs: none. Reason: This candidate was withdrawn by its CNA. Further investigation showed that it was not a security issue. Notes: none"
      }
    ],
    "references": [],
    "metrics": {}
  }
}
